dist/api_data.py

```diff
@@ -721,6 +721,28 @@ def __ge__(self, other):
             this will update the value if one is already set''',
             min='1MB', max='10TB')
         ]),
+    Config('tiered_manager', '', r'''
+        tiered storage manager configuration options''',
+        type='category', subconfig=[
```

Comment:
I think this should be marked `undoc=true`.  I know that one of the APIs has been ifdefed out for doxygen, but we should probably leave these parts of tiering out of the doc as well until we're ready to commit support to them.  (Not sure it's applicable to do here, and/or in `tiered_storage_configuration_common`, but should be undoc for all new APIs.